test/commands/add/tag_spec.rb

```diff
@@ -7,6 +7,15 @@
 
   let(:content) { CONTENT }
 
+  describe "when friend name and tag are blank" do
```

Comment:
Same comment here: my read is that we need both of these to be blank because if only one is blank we'll just be repeating the `"when tag is blank"` test below. Is that right?

Reply:
Yes, I implemented it with that intention.